Code/Editor/CryEditDoc.cpp

```diff
@@ -195,7 +184,7 @@ CCryEditDoc::DocumentEditingMode CCryEditDoc::GetEditMode() const
 
 QString CCryEditDoc::GetActivePathName() const
 {
-    return DocumentEditingMode() == CCryEditDoc::DocumentEditingMode::SliceEdit ? GetSlicePathName() : GetLevelPathName();
+    return GetEditMode() == CCryEditDoc::DocumentEditingMode::SliceEdit ? GetSlicePathName() : GetLevelPathName();
```

Comment:
Just checked the code and this was a great spot! You think there'd be some sort of warning for this! 🤦

Reply:
Can't take credit, Clang tidy spotted this one :)